profile/profile-layered.go

```diff
@@ -165,6 +165,25 @@ func (lp *LayeredProfile) LocalProfile() *Profile {
 	return lp.localProfile
 }
 
+// RevisionCnt returns the current profile revision counter.
+func (lp *LayeredProfile) IncreaseRevisionCounter(lock bool) (revisionCounter uint64) {
+	if lp == nil {
+		return 0
```

Comment:
Comment is wrong and I'm not sure we should really export this method. Having a parameter `lock` that can be specified from outside the package seems error prone

Reply:
Yup, will fix comment and make it internal.